feature/experimental/gribi/otg_tests/fib_failed_due_to_hw_res_exhaust_test/fib_failed_due_to_hw_res_exhaust_test.go

```diff
@@ -0,0 +1,480 @@
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fib_failed_due_to_hw_res_exhaust_test
+
+import (
+	"context"
+	"encoding/binary"
+	"fmt"
+	"net"
+	"testing"
+	"time"
+
+	"github.com/open-traffic-generator/snappi/gosnappi"
+	"github.com/openconfig/featureprofiles/internal/attrs"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/featureprofiles/internal/gribi"
+	"github.com/openconfig/gribigo/fluent"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ondatra/otg"
+	"github.com/openconfig/ygnmi/ygnmi"
+	"github.com/openconfig/ygot/ygot"
+
+	aftspb "github.com/openconfig/gribi/v1/proto/service"
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+// Settings for configuring the baseline testbed with the test
+// topology.
+//
+// The testbed consists of ate:port1 -> dut:port1,
+// dut:port2 -> ate:port2.
+//
+//   * ate:port1 -> dut:port1 subnet 192.0.2.1/30
+//   * ate:port2 -> dut:port2 subnet 192.0.2.5/30
+
+const (
+	dstIPBlock                = "203.0.113.0"
+	vipBlock                  = "198.51.100.0"
+	wantLoss                  = true
+	dutAS                     = 64500
+	ateAS                     = 64501
+	advertisedRoutesv4        = "20.0.1.1"
+	advertisedRoutesv4MaskLen = 32
+	tolerancePct              = 2
+	tolerance                 = 50
+)
+
+var (
+	vendorSpecRoutecount = map[ondatra.Vendor]uint32{
+		ondatra.JUNIPER: 2500000,
+	}
+	dutPort1 = attrs.Attributes{
+		Desc:    "dutPort1",
+		IPv4:    "192.0.2.1",
+		IPv4Len: 30,
+	}
+
+	atePort1 = attrs.Attributes{
+		Name:    "atePort1",
+		IPv4:    "192.0.2.2",
+		MAC:     "02:00:01:01:01:01",
+		IPv4Len: 30,
+	}
+
+	dutPort2 = attrs.Attributes{
+		Desc:    "dutPort2",
+		IPv4:    "192.0.2.5",
+		IPv4Len: 30,
+	}
+
+	atePort2 = attrs.Attributes{
+		Name:    "atePort2",
+		IPv4:    "192.0.2.6",
+		MAC:     "02:00:02:01:01:01",
+		IPv4Len: 30,
+	}
+	fibPassedDstRoute string
+	fibFailedDstRoute string
+)
+
+func configureBGP(dut *ondatra.DUTDevice) *oc.NetworkInstance_Protocol {
+	d := &oc.Root{}
+	ni1 := d.GetOrCreateNetworkInstance(deviations.DefaultNetworkInstance(dut))
+	niProto := ni1.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP")
+	bgp := niProto.GetOrCreateBgp()
+
+	g := bgp.GetOrCreateGlobal()
+	g.As = ygot.Uint32(dutAS)
```

Comment:
could you please below  after line 105 
	```g.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST).Enabled = ygot.Bool(true)
	g.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST).Enabled = ygot.Bool(true)```

Reply:
We only have v4 peer and enabled ipv4 unicast in neighbor level.

Reply:
@cprabha , this was already approved by google that we need to enable AFI_SAFI at global level and neighbour level can be over written , that is how most of bgp scripts are doing  for example we can refer. https://github.com/openconfig/featureprofiles/pull/1171/ .   I request to add above two lines.

Reply:
updated.